datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/datadog_checks/{check_name}/check.py

```diff
@@ -7,5 +7,28 @@
 class {check_class}(AgentCheck):
     def check(self, _):
         # type: (Any) -> None
-        # Use self.instance to read the check configuration
+        # The following are useful bits of code to help new users started.
+
+        # This is how you make HTTP Requests.
+        # More info at https://datadoghq.dev/integrations-core/base/http/
+        # self.http.get("<url>")
+
+        # This is how you submit metrics
+        # self.gauge("test", 1.23, tags=['foo:bar'])
+
+        # There are different types of metrics that you can submit (events, count).
+        # More info at https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.checks.base.AgentCheck
+        # self.count("count", 1.0)
+
+        # This is how you can perform database queries.
+        # You would need to define your query manager in your __init__ method.
```

Comment:
```suggestion
        # Define a query manager in the __init__ method and perform database queries.
```
Maybe this should also contain an __init__ method with signature?